edb/pgsql/delta.py

```diff
@@ -2044,21 +2073,13 @@ def _redo_everything(
 
         cmd = sd.DeltaRoot()
 
-        for prop, new_typ in props:
-            delta_alter, cmd_alter, alter_context = prop.init_delta_branch(
+        for prop, _ in props:
+            delta_alter, cmd_alter, _ = prop.init_delta_branch(
                 schema, context, cmdtype=sd.AlterObject)
-            cmd_alter.set_attribute_value(
-                'target', prop.get_target(orig_schema))
             cmd_alter.set_attribute_value(
```

Comment:
Why does the default update need to get split off?

Reply:
Because the default expression might reference objects that need to be re-created first.